src/partials/auth/login.html

```diff
@@ -31,13 +34,18 @@ <h1 class="text-center">Login</h1>
       <div class="col-sm-4">
         <label>
           <input type="checkbox" name="remember_me" data-ng-model="loginForm.remember_me">
-          Remember Me
+          {{ "AUTH.REMEMBER_ME" | translate }}
```

Comment:
Change to `<span translate>AUTH.REMEMBER_ME</span>` to use directive like in other places